README.md

```diff
@@ -32,6 +32,7 @@ Here are all the steps you need to take as a Boost repository maintainer to enab
 1. Copy the `.azure-pipelines.yml` file from this repository into the top level of your repository.
 1. Copy the `.travis.yml` file from this repository into the top level of your repository.
 1. Copy the `.github/workflows/ci.yml` file from this repository into the the same folder in your repository.
+    * Note that those include CMake tests, so see especially the [CMake docs](CMake.md#ci-tests) for what it expects.
```

Comment:
Suggestions for this sentence:
- at least, remove the word "especially"
- possibly, change the sentence to this:

```
Note that those include CMake tests, see the [CMake docs](CMake.md#ci-tests) for more info.
```

Reply:
alternatives to "info" could be "details", "advice", "requirements", or  "see the CMake docs for further explanation."